etrago/cluster/spatial.py

```diff
@@ -664,7 +676,7 @@ def kmedoids_dijkstra_clustering(etrago, buses, connections, weight, n_clusters)
         )
 
     # k-mean clustering
-    if not settings["k_busmap"]:
+    if not csv:
```

Comment:
I would keep the if statement referring to a variable in "settings" since there could be the case that "csv" is not defined which could lead to an error.

Reply:
csv is the respecting setting (see line 662).
I chaged it, because this function is used like that for AC and gas, so there were misunderstandings in the saving of csv-files. Now, the correct carrier is in the name of the file.